app/src/test/java/com/duckduckgo/app/browser/refreshpixels/RefreshPixelSenderTest.kt

```diff
@@ -143,181 +115,28 @@ class RefreshPixelSenderTest {
     }
 
     @Test
-    fun whenSendMenuRefreshPixelsThenTimeBasedPixelsFired() = runTest {
```

Comment:
Overall a lot of tests have been deleted, are the others covering for the new logic, pixels, etc?

Reply:
yes!